### PR TITLE
Docs: Setup i18n  & Add spanish translation for `docs/index.md`

### PR DESCRIPTION
**Pull request by rafnixg**
- Setup multi-language docs
- Translate index.md (Overview)
- Translate Nav menu

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -41,6 +41,37 @@ plugins:
           rendering:
             show_root_heading: true
             show_source: false
+  - i18n:
+      default_language: en
+      languages:
+        en:
+          name: English
+          build: true
```
Comment (jowilf) [inline mkdocs.yml:49]: I prefer `build=false` here to avoid duplication
```suggestion
          build: false
```

File: docs/index.es.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+- Exportación de datos a CSV/EXCEL/PDF e impresión con navegador
+- Autenticación
+- Autorización
+- Administrar archivos
+- Vistas personalizadas
+- ORMs compatibles
+    * [SQLAlchemy](https://www.sqlalchemy.org/)
+    * [SQLModel](https://sqlmodel.tiangolo.com/)
+    * [MongoEngine](http://mongoengine.org/)
+    * [ODMantic](http://mongoengine.org/)
```
Comment (jowilf) [inline docs/index.es.md:46]: This link is not the good one. I will update for english version
```suggestion
    * [ODMantic](hhttps://github.com/art049/odmantic/)
```

**Review by jowilf — CHANGES_REQUESTED** [review]
Really thank you for this work, you are doing great.

For more readability in long term (when maybe we will have a lot of languages), i think we should move each page inside a directory. Otherwise we should use folder based docs structure . But  I prefer suffix based docs structure with each page inside a directory

So, the new location for `index.md` and `index.es.md` should be `docs/index/index.md` and `docs/index/index.es.md`

**Comment by rafnixg** [comment]
> Really thank you for this work, you are doing great.
> 
> For more readability in long term (when maybe we will have a lot of languages), i think we should move each page inside a directory. Otherwise we should use folder based docs structure . But I prefer suffix based docs structure with each page inside a directory
> 
> So, the new location for `index.md` and `index.es.md` should be `docs/index/index.md` and `docs/index/index.es.md`

I already made the change to the new folder structure, the file index.md and index.es.md are the only ones that are outside a folder, so that Mkdocs recognizes the entry point

**Review by jowilf — APPROVED** [review]

**Comment by jowilf** [comment]
Thank you for your contribution

**Comment by rafnixg** [comment]
in the next few days I will translate what is missing, thanks to you for this library!

**Pull request merged**